### DISCUSSION TITLE
有通配符目标吗?

### DISCUSSION THREAD

**Discussion by ghost** [xmake-io/xmake#2803] [Q&A]
### 你在什么场景下需要该功能？

因为经常需要写各种各样的测试demo,做C++技术验证. 需要写很多target, 时常加一些新目标太麻烦了.
我一直用的是Makefile因为他支持通配符目标. 但Makefile也并不完美,也有缺点.
xmake支持通配符吗?

### 描述可能的解决方案

支持通配符*和?,模式匹配.

**Comment by waruqi** [comment]
自己 for-loop  os.files 批量加一下 也没几行代码  https://github.com/idealvin/coost/blob/c6ced3cbe20afcdff4cbddba78d7b2d5ecc57de2/test/xmake.lua#L3
